pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-caching.adoc

```diff
@@ -0,0 +1,46 @@
+= Sharing Data Between Workers and Caching
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+NOTE: To view an example policy project that uses caching, see https://github.com/mulesoft/pdk-custom-policy-examples/tree/main/data-caching/README.md[Data Caching Policy Example^].
+
+Envoy spawns many workers for the policy to increase the capability of handling requests. A single worker handles all the stages of a specific request, including outgoing connections and the response flow, enabling you to track an internal status for a specific request and create a sequential execution flow even though the underlying implementation is event-driven. 
+
+Each worker processes a single request at a given time. To share a global state in a specific worker, use a `RefCell` without any risk of concurrent modification.
```

Comment:
As a newbie, I'm not sure what a `RefCell` is. Is it a parameter in some file? Would it be self-explanatory to a new user? I don't see an example of it in the below snippets. Or can you link to an explanation if explained on another page?